playkit/src/main/java/com/kaltura/playkit/player/ExoPlayerWrapper.java

```diff
@@ -481,6 +482,13 @@ public void onLoadingChanged(boolean isLoading) {
         log.d("onLoadingChanged. isLoading => " + isLoading);
     }
 
+    @Override
+    public void onIsPlayingChanged(boolean isPlaying) {
+        if (isPlaying) {
+            sendDistinctEvent(PlayerEvent.Type.PLAYING);
```

Comment:
@GouravSna 
is there any change in the behaviour if PLAYING event due to this update?

Reply:
From event prospective PLAYING is being sent as usual.